version.go

```diff
@@ -0,0 +1,22 @@
+package triton
+
+import (
+	"fmt"
+	"runtime"
+)
+
+// The main version number of the current released Triton-go SDK.
+const Version = "1.0.0"
+
+// A pre-release marker for the version. If this is "" (empty string)
+// then it means that it is a final release. Otherwise, this is a pre-release
+// such as "dev" (in development), "beta", "rc1", etc.
+var Prerelease = "dev"
```

Comment:
Can you include `go build` linker instructions to denote a release?  I'd rather flip this around so that it's not `"dev"` by default because this is a library.  If something wants to add "dev" to the useragent because the consuming application is still under development, that makes sense to me.  But doing it here seems like it'll be a PITA to root through the code to change.  My worry being that everything will be decorated with `triton-go/1.0.0-dev` unless they dig through and provide the right linker incantation to swap out this variable (e.g. think about what'd be involved in changing this in the Terraform provider).

Reply:
I don't know go build linker but i will remove the string for now :)